src/lib/local-storage-utils.js

```diff
@@ -0,0 +1,53 @@
+/* @flow */
+import constants from './constants';
+import generate from './generate-id';
+
+const { storage, sevenDays } = constants;
+
+/* Returns an existing cartId or null */
+export const getCartId = () => {
+    const storedValue = window.localStorage.getItem(storage.paypalCrCart);
```

Comment:
this function should return `valid existing cartId`, not just cartId right? I don't think it should be developer logic to handle validity of cartIds.